functions.php

```diff
@@ -444,11 +444,11 @@ function get_project_status($post_id) {
 
     $mypod = pods('projects', $post_id);
 
-    $intake_status = $mypod -> field('intake_status_date');
```

Comment:
Changing the field means a change in the PODS setup. The old intake date that was on things might get lost, right?